warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/edge/ProtobufEdgeDataTypeHandler.java

```diff
@@ -698,6 +741,36 @@ public long process(KEYIN key, RawRecordContainer event, Multimap<String,Normali
         return edgesCreated;
     }
     
+    private void setupPreconditionEvaluation(Multimap<String,NormalizedContentInterface> fields) {
+        long start = System.currentTimeMillis();
+        edgePreconditionContext.setFilteredContextForNormalizedContentInterface(fields);
+        edgePreconditionEvaluation.setJexlContext(edgePreconditionContext);
+        if (log.isTraceEnabled()) {
+            long time = System.currentTimeMillis() - start;
+            // only worth logging those that took some time....
+            if (time > 1) {
+                if (log.isTraceEnabled()) {
```

Comment:
These if statements can be collapsed into 
```
if (time > 1 && log.isTraceEnabled()) {
```
Thanks for cleaning up other people's code and makign it more readable.